### PR TITLE
Add lmp-support layer

### PR DESCRIPTION
**Pull request by esajaa**
LmP (Linux microPlatform) is a Yocto-based Linux platform. This PR adds support to that platform. Also, utilizing the BSP from the LmP, this PR should enable Pelion Edge to run on i.MX 8M Mini Evaluation Kit and Avnet UltraZed-EG

The support is added with an additional meta-lmp-support layer, that can be included or excluded from the build as required using the bblayers.conf. The layer includes image-recipe, fstabs & custom partition layout. For the i.MX board there are patches to make the image run from SD-card. For the Avnet board there's an additional kernel configuration for CPU stability.  Also, there's a fluentbit compilation issue fix for distros that have usrmerge feature endabled.

### CODE REVIEW

**Review by JanneKiiskila — APPROVED** [review]
LGTM

**Review by ygoyal18 — APPROVED** [review]
LGTM

**Review by x85446 — APPROVED** [review]
Looks good. I agree with your approach of putting the meta inside the meta-pelion-edge.  In the next release we should continue along this theme and consolidate other meta's to the top level of meta-pelion-edge.  (e.g.) meta-pe-containers

File: meta-lmp-support/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Meta for Linux microplatform (LmP)
+
+This meta-layer contains required support recipes to build and run the Pelion Edge on LmP, a Linux based platform from [Foundries.io](https://foundries.io/).
+
+For more information, please see the [Pelion Edge documentation](https://www.pelion.com/docs/device-management-edge/latest/introduction/index.html), and [the quickstart for LmP](https://www.pelion.com/docs/device-management-edge/latest/quick-start/lmp-quick-start.html)
```
Comment (cmonr) [inline meta-lmp-support/README.md:5]: @AnotherButler Does this kind of documentation need to be on your radar?

Comment (esajaa) [reply meta-lmp-support/README.md:5]: This documentation will be added in dm-docs PR 102

**Pull request merged**